data-prepper-plugins/saas-source-plugins/jira-source/build.gradle

```diff
@@ -0,0 +1,31 @@
+plugins {
+    id 'java'
+}
+
+dependencies {
+
+    implementation project(path: ':data-prepper-plugins:saas-source-plugins:saas-crawler')
+    implementation project(path: ':data-prepper-api')
+    implementation project(path: ':data-prepper-plugins:aws-plugin-api')
+    implementation project(path: ':data-prepper-plugins:buffer-common')
+    implementation project(path: ':data-prepper-plugins:common')
+
+    implementation libs.commons.io
+    implementation 'io.micrometer:micrometer-core'
+    implementation 'com.fasterxml.jackson.core:jackson-core'
+    implementation 'com.fasterxml.jackson.core:jackson-databind'
+    implementation 'javax.inject:javax.inject:1'
+    implementation 'org.springframework:spring-web:5.3.39'
+    implementation 'org.springframework.retry:spring-retry:1.3.4'
```

Comment:
Do we need this dependency? I don't see it in use.

Reply:
Like we discussed, I am relaying on this library in the code. I removed in this PR and I will add it back when I add jira source code 👍